pkg/config/repoconfig/repoconfig_suite_test.go

```diff
@@ -0,0 +1,86 @@
+package repoconfig
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("repoconfig parsing", func() {
+	When("the yaml file has no label-approved", func() {
+		It("should return no LabelsApproved", func() {
+			config := []byte("")
+			botConfig, err := parseConfigYAML(config)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(botConfig.LabelApproved()).To(BeNil())
```

Comment:
There was a comment here which was cleared by github.

I'm not changing behavior with this patch, just documenting / testing the existing behavior. I know it's not what you intended, but it's what the yaml parser is giving:

if the yaml is

```
label-approved:
```

Then the LabelApproved field turns to be nil... :-/